diff --git a/node_modules/@react-navigation/native/lib/commonjs/withOrientation.js b/node_modules/@react-navigation/native/lib/commonjs/withOrientation.js
index 324c382..8b1913a 100644
--- a/node_modules/@react-navigation/native/lib/commonjs/withOrientation.js
+++ b/node_modules/@react-navigation/native/lib/commonjs/withOrientation.js
@@ -51,11 +51,11 @@ function _default(WrappedComponent) {
     }
 
     componentDidMount() {
-      _reactNative.Dimensions.addEventListener('change', this.handleOrientationChange);
+      this.dimensionsSubscription = Dimensions.addEventListener('change', this.updateWindowDims); 
     }
 
     componentWillUnmount() {
-      _reactNative.Dimensions.removeEventListener('change', this.handleOrientationChange);
+      this.dimensionsSubscription?.remove();
     }
 
     render() {
diff --git a/node_modules/@react-navigation/native/lib/module/withOrientation.js b/node_modules/@react-navigation/native/lib/module/withOrientation.js
index fbc1c0b..88d8630 100644
--- a/node_modules/@react-navigation/native/lib/module/withOrientation.js
+++ b/node_modules/@react-navigation/native/lib/module/withOrientation.js
@@ -31,11 +31,12 @@ export default function (WrappedComponent) {
     }
 
     componentDidMount() {
-      Dimensions.addEventListener('change', this.handleOrientationChange);
+      this.dimensionsSubscription = Dimensions.addEventListener('change', this.updateWindowDims);
+   
     }
 
     componentWillUnmount() {
-      Dimensions.removeEventListener('change', this.handleOrientationChange);
+      this.dimensionsSubscription?.remove();
     }
 
     render() {
diff --git a/node_modules/@react-navigation/native/src/withOrientation.js b/node_modules/@react-navigation/native/src/withOrientation.js
index c449653..e60a7c9 100644
--- a/node_modules/@react-navigation/native/src/withOrientation.js
+++ b/node_modules/@react-navigation/native/src/withOrientation.js
@@ -14,11 +14,11 @@ export default function (WrappedComponent) {
     }
 
     componentDidMount() {
-      Dimensions.addEventListener('change', this.handleOrientationChange);
+      this.dimensionsSubscription = Dimensions.addEventListener('change', this.updateWindowDims);
     }
 
     componentWillUnmount() {
-      Dimensions.removeEventListener('change', this.handleOrientationChange);
+      this.dimensionsSubscription?.remove();
     }
 
     handleOrientationChange = ({ window }) => {
