diff --git a/node_modules/react-native-mumeng/android/src/main/java/com/maochunjie/mumeng/RNReactNativeMumengModule.java b/node_modules/react-native-mumeng/android/src/main/java/com/maochunjie/mumeng/RNReactNativeMumengModule.java
index 302eb1e..8dc4690 100644
--- a/node_modules/react-native-mumeng/android/src/main/java/com/maochunjie/mumeng/RNReactNativeMumengModule.java
+++ b/node_modules/react-native-mumeng/android/src/main/java/com/maochunjie/mumeng/RNReactNativeMumengModule.java
@@ -140,6 +140,11 @@ public class RNReactNativeMumengModule extends ReactContextBaseJavaModule {
         MobclickAgent.onEventValue(reactContext, eventId, rMap, value);
     }
 
+    @ReactMethod
+	public void reportError(String content) {
+    	MobclickAgent.reportError(reactContext, content);
+	}
+
     /********************************U-App统计*********************************/
 
     public static String getDeviceInfo(Context context) {
@@ -274,4 +279,4 @@ public class RNReactNativeMumengModule extends ReactContextBaseJavaModule {
     public static void onResume(Context context) {
         MobclickAgent.onResume(context);
     }
-}
\ No newline at end of file
+}
diff --git a/node_modules/react-native-mumeng/index.js b/node_modules/react-native-mumeng/index.js
index 9626e4e..779db17 100644
--- a/node_modules/react-native-mumeng/index.js
+++ b/node_modules/react-native-mumeng/index.js
@@ -78,3 +78,7 @@ export function onEventWithMap(eventId, eventMap) {
 export function onEventWithMapAndCount(eventId, eventMap, eventCount) {
     RNReactNativeMumeng.onEventWithMapAndCount(eventId, eventMap, eventCount);
 }
+
+export function reportError(content) {
+	RNReactNativeMumeng.reportError(content);
+}
diff --git a/node_modules/react-native-mumeng/ios/RNReactNativeMumeng.m b/node_modules/react-native-mumeng/ios/RNReactNativeMumeng.m
index d7e1edf..d99a41a 100644
--- a/node_modules/react-native-mumeng/ios/RNReactNativeMumeng.m
+++ b/node_modules/react-native-mumeng/ios/RNReactNativeMumeng.m
@@ -89,7 +89,7 @@ - (dispatch_queue_t)methodQueue
   if (parameters == nil && [parameters isKindOfClass:[NSNull class]]) {
     parameters = nil;
   }
-  
+
   [MobClick event:eventId attributes:parameters counter:eventNum];
 }
 
@@ -109,5 +109,10 @@ - (dispatch_queue_t)methodQueue
   [MobClick endLogPageView:pageName];
 }
 
+RCT_EXPORT_METHOD(reportError:(NSString *)content)
+{
+  [MobClick reportError:content];
+}
+
 @end
-  
+
