diff --git a/node_modules/react-native-9patch-image/android/build.gradle b/node_modules/react-native-9patch-image/android/build.gradle
index 28ce2d1..5a40e9f 100644
--- a/node_modules/react-native-9patch-image/android/build.gradle
+++ b/node_modules/react-native-9patch-image/android/build.gradle
@@ -26,7 +26,7 @@ android {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+   implementation 'com.facebook.react:react-native:+'
 }
 
 repositories {
diff --git a/node_modules/react-native-9patch-image/android/src/main/java/su/rusfearuth/reactnative/native9patch/RCTImageCache.java b/node_modules/react-native-9patch-image/android/src/main/java/su/rusfearuth/reactnative/native9patch/RCTImageCache.java
index 95cde1c..5681803 100644
--- a/node_modules/react-native-9patch-image/android/src/main/java/su/rusfearuth/reactnative/native9patch/RCTImageCache.java
+++ b/node_modules/react-native-9patch-image/android/src/main/java/su/rusfearuth/reactnative/native9patch/RCTImageCache.java
@@ -1,7 +1,7 @@
 package su.rusfearuth.reactnative.native9patch;
 
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 
 import java.lang.ref.WeakReference;
 import java.util.HashMap;
diff --git a/node_modules/react-native-9patch-image/android/src/main/java/su/rusfearuth/reactnative/native9patch/RCTImageCapInsetView.java b/node_modules/react-native-9patch-image/android/src/main/java/su/rusfearuth/reactnative/native9patch/RCTImageCapInsetView.java
index 2adca36..f07fe4c 100644
--- a/node_modules/react-native-9patch-image/android/src/main/java/su/rusfearuth/reactnative/native9patch/RCTImageCapInsetView.java
+++ b/node_modules/react-native-9patch-image/android/src/main/java/su/rusfearuth/reactnative/native9patch/RCTImageCapInsetView.java
@@ -1,7 +1,7 @@
 package su.rusfearuth.reactnative.native9patch;
 
 import android.content.Context;
-import android.support.annotation.NonNull;
+import androidx.annotation.NonNull;
 import android.widget.ImageView;
 
 import su.rusfearuth.reactnative.native9patch.RCTImageCache;
